Dockerfile

```diff
@@ -1,5 +1,5 @@
 # ===============================================
-FROM helsinkitest/node:14-slim as appbase
+FROM helsinkitest/node:18-slim as appbase
```

Comment:
18 or 20?

Reply:
It's 18 until we have helsinkitest/node:20 available. 18 should work ok.